models/src_zuora.yml

```diff
@@ -7,9 +7,9 @@ sources:
     loader: fivetran
     loaded_at_field: _fivetran_synced
 
-    freshness:
-      warn_after: { count: 72, period: hour }
-      error_after: { count: 168, period: hour }
+    freshness: 
+      warn_after: { count: 72, period: hour } 
+      error_after: { count: 168, period: hour } 
```

Comment:
Good question. I replicated the Zuora freshness tests from Recurly since the use case was similar (subscription models usually will not need daily updates so we can extend it out to biweekly/weekly checks). I think for now global makes sense unless we think item tables need to be checked more frequently?